content-guidelines/extension-title.md

```diff
@@ -8,9 +8,9 @@ skip_index: true
 
 # Extension Name
 
-Your extension’s name is critical. It shows up everywhere—in search results, on [addons.mozilla.org](https://addons.mozilla.org 'addons.mozilla.org'), and in the browser itself. It’s sometimes one of the few pieces of information that a user has to determine whether or not they are interested in what you built.
+Your extension’s name is critical. It shows up everywhere—in search results, on [addons.mozilla.org][https://addons.mozilla.org 'addons.mozilla.org'], and in the browser itself. It’s sometimes one of the few pieces of information that a user has to determine whether or not they are interested in what you built.
 
-Research shows that people really do take an extension’s name into account when making installation decisions. And, what you put into extension name field becomes your URL on [addons.mozilla.org](https://addons.mozilla.org 'addons.mozilla.org').
+Research shows that people really do take an extension’s name into account when making installation decisions. And, what you put into extension name field becomes your URL on [addons.mozilla.org][https://addons.mozilla.org 'addons.mozilla.org'].
```

Comment:
Same here and every other places

Reply:
@championshuttler  creating a key like
 [addons.mozilla.org] : https://addons.mozilla.org 'addons.mozilla.org'
 OR 
 [duplicate link]: https://addons.mozilla.org 'addons.mozilla.org' towards the end of  markdown file  be a good resolution?